### DISCUSSION TITLE
Companion app with GUI

### DISCUSSION THREAD

**Discussion by denizsafak** [smacke/subsync#200] [Show and tell]
Hello everyone, I've been working on a companion app for ffsubsync and alass called [AutoSubSync](https://github.com/denizsafak/AutoSubSync). It includes several additional features, such as:

- Support for different subtitle formats (.srt, .vtt, .sbv, .sub, .ass, .ssa, .dfxp, .ttml, .itt, .stl)
- Batch processing to handle multiple video and subtitle pairs.
- “Auto-Pairing with Season/Episode” for pairing videos and reference subtitles with subtitles automatically using Season/Episode patterns like S01E01, 1x01, etc.
- Manual sync for fine-tuning the subtitles.
- A user-friendly GUI.

> [!NOTE]
> I haven't tested the MacOS build. If you are able to test it, please let me know if it works. "alass" probably will not work in MacOS. If that happens, try `brew install alass` and check again. If does not work again, please open an issue.

Also, many thanks to @smacke for making this increadible tool.

**Comment by farebyting** [comment]
Thanks for make this :D
